components/storage/blockdevice/COMPONENT_QSPIF/QSPIFBlockDevice.cpp

```diff
@@ -874,6 +875,7 @@ int QSPIFBlockDevice::_sfdp_set_quad_enabled(uint8_t *basic_param_table_ptr)
             status_reg_setup[1] = 0x2; // Bit 1 of status Reg 2
             _read_register_inst = 0x35;
             sr_read_size = 1;
+	    sr_write_size = 2;
```

Comment:
seems to be misaligned (astyle should catch these)

Reply:
sorry fixing